src/tools/networkTools.cpp

```diff
@@ -212,39 +212,38 @@ std::map<std::string, std::string> getNetworkInterfaces() {
 }
 
 
-std::string getBestPublicIp(bool ipv6) {
-  IpAddress bestPublicIp = IpAddress{"127.0.0.1","::1"};
+IpAddress getBestPublicIps() {
+  IpAddress bestIp = IpAddress{"127.0.0.1","::1"};
   std::map<std::string, IpAddress> interfaces = getNetworkInterfacesIPv4Or6();
 
 #ifndef _WIN32
   const char* const prioritizedNames[] =
       { "eth0", "eth1", "wlan0", "wlan1", "en0", "en1" };
   for(auto name: prioritizedNames) {
     auto it=interfaces.find(name);
-    if(it != interfaces.end() && !(ipv6 && (*it).second.addr6.empty())) {
-      bestPublicIp = (*it).second;
-      break;
+    if (it != interfaces.end()) {
+      if (it->second.addr.find(':') == std::string::npos) bestIp.addr = it->second.addr;
+      else bestIp.addr6 = it->second.addr6;
     }
```

Comment:
Since the `break` statement has disappeared. Does that mean that what was previously considered the best public IP address may now be overwritten by continued execution of the loop?